### ISSUE TITLE
Linebreaks?

### ISSUE THREAD

**Issue by Sajjon** [apple/swift-book#18]
Currently the content contains a lot of manual line breaks.

Take a look at the rendered first paragraph of the [current website](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html):

<img width="766" alt="Screenshot 2022-08-30 at 21 04 31" src="https://user-images.githubusercontent.com/864410/187521443-54b4fec1-5091-4da9-800f-968018fa6c85.png">

And now take a look at the content source:

```text
Swift is a new programming language for iOS, macOS, watchOS, and tvOS app development.
Nonetheless, many parts of Swift will be familiar
from your experience of developing in C and Objective-C.
```

Or more clear by looking at it in Xcode, where line numbers are written our:
<img width="796" alt="Screenshot 2022-08-30 at 21 05 36" src="https://user-images.githubusercontent.com/864410/187521646-ca95339f-eb2d-4a10-ba6c-8dbe9431ab46.png">

Having to think about inserting line breaks manually to make it render nicely is cumbersome. It breaks flow when typing and surely it is a presentation matter? Also, it prevents dynamic width of the rendered book!

I suggest we rid the book of these manual line breaks and allow large screens to make use of its pixels! 
<img width="1510" alt="Screenshot 2022-08-30 at 21 07 48" src="https://user-images.githubusercontent.com/864410/187522120-a84ce94d-39b4-4317-8f89-6528c0f75879.png">

To summarise, my suggestion has two advantages:
1. Writing the book becomes easier
2. Rendering of the book becomes more powerful because we can use the full width of screens.

**Comment by amartini51** [comment]
The manual line breaks in source make diffing and merging easier in Git — for more detail, please see the [style guide](https://github.com/apple/swift-book/blob/main/Style.md).  Both the legacy RST-based publication pipeline and the DocC pipeline use blank lines to separate paragraphs, so these manual line breaks have no effect on the published content.

**Comment by krilnon** [comment]
I'll echo Alex here; this writing style has been really useful for reviewing content changes. It doesn't change the rendered output, so it's something we should discuss as part of the writing process.

**Comment by Sajjon** [comment]
Thx for clarification! I just have to get used to entering.. enter! :)
